docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,177 @@
+```
+
+To deploy Kyma using your own Kyma installer image, run:
+
+```bash
+kyma alpha deploy --source {IMAGE}
+```
+
+To build an image from your local sources and deploy Kyma on a remote cluster based on this image, run:
+
+```bash
+kyma alpha deploy --source local --custom-image {IMAGE}
+```
+>    **EXAMPLE:** If your `{IMAGE}` is `user/my-kyma-installer:v1.4.0`, this command builds an image from your local sources and push it your repository `user/my-kyma-installer` with the tag `v1.4.0`.<br>
+Then, this image `user/my-kyma-installer:v1.4.0` is used in the `Deployment` for `kyma-installer`.
+
+To deploy kyma with only specific components, run:
+
+```bash
+kyma alpha deploy --components {COMPONENTS_FILE_PATH}
+```
+`{COMPONENTS_FILE_PATH}` is the path to a YAML file containing the desired component list to be installed, for example:
+```bash
+  components:
+  - name: "cluster-essentials"
+    namespace: "kyma-system"
+  - name: "testing"
+    namespace: "kyma-system"
+  - name: "istio"
+    namespace: "istio-system"
+  - name: "xip-patch"
+    namespace: "kyma-installer"
+  - name: "istio-kyma-patch"
+    namespace: "istio-system"
+  - name: "dex"
+    namespace: "kyma-system"
+  ```
+In this example, only these 6 components are deployed on the cluster.
+
+To override the standard Kyma installation, run:
+<!---didn't find override among the deploy options--->
+```bash
+kyma alpha deploy --override {OVERRIDE_FILE_PATH}
+```
+`{OVERRIDE_FILE_PATH}` is the path to a YAML file containing the parameters to override, for example:
+```bash
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ory-overrides
+    namespace: kyma-installer
+    labels:
+      installer: overrides
+      component: ory
+      kyma-project.io/installation: ""
+  data:
+    hydra.deployment.resources.limits.cpu: "153m"
+    hydra.deployment.resources.requests.cpu: "53m"
+  ---
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: monitoring-overrides
+    namespace: kyma-installer
+    labels:
+      installer: overrides
+      component: monitoring
+      kyma-project.io/installation: ""
+  data:
+    alertmanager.alertmanagerSpec.resources.limits.memory: "304Mi"
+    alertmanager.alertmanagerSpec.resources.requests.memory: "204Mi"
+  ```
+In this example, overrides are provided for 2 different components: `ory` and `monitoring`.
+ 
+- For `ory`, the values of `hydra.deployment.resources.limits.cpu` and `hydra.deployment.resources.requests.cpu` will be overriden to `153m` and `53m` respectively.
+- For `monitoring`, the values of `alertmanager.alertmanagerSpec.resources.limits.memory` and `alertmanager.alertmanagerSpec.resources.requests.memory` will be overriden to `304Mi` and `204Mi` respectively.
+
+It is also possible to provide multiple override files at the same time
+
+  ```bash
+  kyma install --override {OVERRIDEL_FILE_1_PATH} --override {OVERRIDE_FILE_2_PATH}
+  ```
+
+## Upgrade Kyma
+<!---upgrade is not among the alpha commands--->
+
```

Comment:
The upgrade will also do a compatibility check between the new and the currently installed version. We should mention, that this version check can only deliver reliable feedback if a Kyma release was installed (not from a PR, branch, revision or local).

Reply:
That's good to know! Added it.